### DISCUSSION TITLE
GPULink python API for 2D data

### DISCUSSION THREAD

**Discussion by zhiyang-fu** [astra-toolbox/astra-toolbox#391] [Q&A]
Is possible to use the existing GPULink wrapper for 2D data? If not, could you @wjp give some guidance on writing custom functions/codes to link 2D gpu data other than the already supported numpy arrays.

Thanks,
Zhiyang

**Comment by Chilie** [comment]
I now also want to implement such function. There is a GPUlink for data3d. How we can write the GPU support for data2d?

**Comment by askorikov** [comment]
As a workaround, you can use `GPULink` with z = 1.
